### ISSUE TITLE
README.md should include basic build instructions

### ISSUE THREAD

**Issue by cgarwood82** [chennes/Stop_Motion_Animation#42]
Was looking to build this and check it out, but it's missing build instructions in the readme.

**Comment by chennes** [comment]
Good idea -- I've added some rudimentary instructions, with the caveat that this development was done on Windows so it's pretty likely that you'll need some experience with QMake to get those FFmpeg libraries to link on Linux or OS X. Feel free to shoot me a pull request if you've got better instructions for those platforms.
